speaker/Dockerfile

```diff
@@ -7,4 +7,8 @@ ADD speaker /speaker
 # just in case
 RUN chmod a+x /speaker
 
+# Production image
+FROM gcr.io/distroless/static:latest
+COPY --from=intermediate /speaker /speaker
+
```

Comment:
Any reason the `USER` directive was there but not here?

Reply:
The image runs as root due to the required capabilities. I was thinking about do I need to indicate the intention here as well I did for `controller` or not - I mean the directive `USER 0`?

Reply:
Well, user 0 is not really required but the capabilities. I think not specifying it here makes sense, then. Thanks! :)